settings/templates/help.php

```diff
@@ -2,26 +2,26 @@
 	<ul>
 	<?php if($_['admin']) { ?>
 		<li>
-			<a class="<?php p($_['style1']); ?>"
-				href="<?php print_unescaped($_['url1']); ?>">
+			<a class="icon-category-office"
```

Comment:
```suggestion
			<a class="icon-category-office <?php p($_['style1']); ?>"
```

to keep the active state